source/language/delays.rst

```diff
@@ -125,14 +125,17 @@ these side effects. Also contrary to TeX, we prohibit overlapping gates.
 Operations on durations
 -----------------------
 
-We can add/subtract two durations, or multiply them by a constant, to get new
-duration. The result must be positive. These are compile time operations since ultimately all
+We can add/subtract two durations, or multiply or divide them by a constant, to get a new
+duration. Division of two durations results in a machine-precision float 
+(see :ref:`divideDuration`). Negative durations are allowed, however
+passing a negative duration to a ``gate[duration]`` or ``box[duration]`` expression will result in an error.
+All durations are compile time operations since ultimately all
```

Comment:
```suggestion
All operations on durations happen at compile time since ultimately all
```

Reply:
Updated in https://github.com/Qiskit/openqasm/pull/268/commits/b3f3b65e875b47cb329e6c2af1fcb4b6601f8fcf